### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by daoanhhuy26012001**

### CODE REVIEW

File: practice-two/src/index.html [hunk 1]
```diff
@@ -192,5 +191,61 @@ <h2 class="started-title">Get started today!</h2>
...
+            </ul>
+            <ul class="list-group">
+              <p class="list-group-item">Contact us</p>
+              <li class="list-group-item">Slack</li>
+              <li class="list-group-item">Jobs</li>
+            </ul>
+          </div>
+        </div>
+      </div>
+      <div class="container-end">
```
Comment (phuocduong-agilityio) [inline practice-two/src/index.html:235]: Use another name

File: practice-two/src/assets/styles/layouts/_footer.scss [hunk 2]
```diff
@@ -0,0 +1,139 @@
+.footer {
+  .container {
+    margin-top: 50px;
+    padding-bottom: 23px;
```
Comment (khanhnguyen-agilityio) [inline practice-two/src/assets/styles/layouts/_footer.scss:4]: @daoanhhuy26012001 Make sure that we need both margin-top and padding-bottom.

**Comment by khanhnguyen-agilityio** [comment]
@daoanhhuy26012001 
 Your result is no match with the design. 


image

File: practice-two/src/assets/styles/layouts/_footer.scss [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+      &::after {
+        padding: 1px 544px;
+        left: 176px;
+      }
+
+      .list-content {
+        padding-right: 257px;
+        .list-box {
+          .list-group {
```
Comment (khanhnguyen-agilityio) [inline practice-two/src/assets/styles/layouts/_footer.scss:122]: @daoanhhuy26012001 Both `list-box` and `list-group` without properties. Can we remove them?

File: practice-two/src/index.html [hunk 4]
```diff
@@ -192,5 +191,61 @@ <h2 class="started-title">Get started today!</h2>
...
+            class="brand-logo"
+            src="./assets/images/headers/logo.svg"
+            alt="logo"
+            onclick="window.location.href='index.html'"
+          />
+        <div class="list-content">
+          <div class="list-box">
+            <ul class="list-group">
+              <p class="list-group-item">Features</p>
+              <li class="list-group-item">Pricing</li>
```
Comment (khanhnguyen-agilityio) [inline practice-two/src/index.html:206]: @daoanhhuy26012001 Missed tag a for all items.

File: practice-two/src/index.html [hunk 5]
```diff
@@ -192,5 +191,61 @@ <h2 class="started-title">Get started today!</h2>
...
+            </ul>
+          </div>
+        </div>
+      </div>
+      <div class="container-end">
+        <p class="details">
+          NinjaMail is a sample project for Quest AI. © 2019 Quest AI, All
+          rights reserved.
+        </p>
+        <div class="link">
```
Comment (khanhnguyen-agilityio) [inline practice-two/src/index.html:240]: @daoanhhuy26012001 Class name is too general.

File: practice-two/src/assets/styles/layouts/_footer.scss [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+      gap: 125px;
+
+      .brand-logo {
+        margin-left: 228px;
+        width: 197px;
+        height: 60px;
+      }
+
+      &::after {
+        padding: 1px 544px;
```
Comment (khanhnguyen-agilityio) [inline practice-two/src/assets/styles/layouts/_footer.scss:115]: @daoanhhuy26012001 Padding with  544px  value is unusual. Please help to check it.

File: practice-two/src/assets/styles/layouts/_footer.scss [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+        left: 176px;
+      }
+
+      .list-content {
+        padding-right: 257px;
```
Comment (khanhnguyen-agilityio) [inline practice-two/src/assets/styles/layouts/_footer.scss:120]: @daoanhhuy26012001 padding-right is too max. pls check it again.

**Pull request merged**